martin/src/srv/server.rs

```diff
@@ -407,6 +413,17 @@ pub fn router(cfg: &mut web::ServiceConfig) {
 
 /// Create a new initialized Actix `App` instance together with the listening address.
 pub fn new_server(config: SrvConfig, all_sources: AllSources) -> crate::Result<(Server, String)> {
+    // metrics setup
+    let labels = HashMap::new();
+
+    // labels.insert("label_foo".to_string(), "value_barbarbbabbmiles".to_string());
+    let prometheus = PrometheusMetricsBuilder::new("martin")
+        .endpoint("/metrics")
```

Comment:
> if we start handling `/metrics`, we would have to declare this as a reserved keyword (we might be missing docs in a martin book for this). One workaround perhaps would be to add everything "magical" to the `/_/metrics` style prefixes because `_` is not a valid source ID i think.
> 
> https://github.com/maplibre/martin/blob/a2acd1732ecdd9ded7ce5fade91518119790e9a6/martin/src/srv/server.rs#L32-L37

Right, as you wish, I'm fine with both solutions (the reserved keyword and the /_/metrics workaround). We just have to keep in mind that `/metrics` is the standard path and the default, so the second solution would mean a little bit more config on the scraper side.

Reply:
good point about conventions (actually, didn't we talk about it elsewhere?), i'm ok to add `/metrics` to reserved then